WebDriverAgentLib/Utilities/FBScreenshot.m

```diff
@@ -70,10 +71,18 @@ + (NSData *)takeWithQuality:(NSUInteger)quality
 {
   if ([self.class isNewScreenshotAPISupported]) {
     XCUIScreen *mainScreen = XCUIScreen.mainScreen;
+
+    // Note to fit to the old spec
```

Comment:
A more descriptive comment would be:

The incoming rect dimensions are already scaled, so we need to adjust them to get a proper screenshot

Reply:
this note was for my working note, haha